pkg/packagekit/autoupdates.jsx

```diff
@@ -270,19 +270,19 @@ const AutoUpdatesDialog = ({ backend }) => {
             <Form isHorizontal onSubmit={save}>
                 <FormGroup fieldId="type" label={_("Type")} hasNoPaddingTop>
                     <Radio isChecked={!enabled}
-                           onChange={e => { setEnabled(false); setType(null) }}
+                           onChange={() => { setEnabled(false); setType(null) }}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)